app/components/nav-bars/top-navbar.js

```diff
@@ -1,10 +1,11 @@
 import Component from '@ember/component';
-
+import { inject as service } from '@ember/service';
 
 export default Component.extend({
-
+  store: service(),
   tagName: 'nav',
   elementId: 'top-navbar',
-  classNames: ['navbar navbar-dark bg-dark navbar-expand-xl']
+  classNames: ['navbar navbar-dark bg-dark navbar-expand-xl'],
+
```

Comment:
mmmm... clean prs mean clean commits. delete the extra line doesn't add anything.